### PR TITLE
Add details around auth0 setup

### PR DESCRIPTION
**Pull request by enorms**
Based on gitter discussion

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-AUTH0_API_AUTHORITY=https://a-domain.us.auth0.com/
-AUTH0_API_AUDIENCE=https://an.audience.domain/
+AUTH0_API_AUTHORITY=https://[AUTH0_DOMAIN]/
+AUTH0_API_AUDIENCE=https://[your-audience-domain]/
 
-AUTH0_AUTHORIZATION_URL=https://a-domain.us.auth0.com/authorize
-AUTH0_TOKEN_URL=https://a-domain.us.auth0.com/oauth/token
+AUTH0_AUTHORIZATION_URL=https://[AUTH0_DOMAIN]/authorize
+AUTH0_TOKEN_URL=https://[AUTH0_DOMAIN]/oauth/token
```
Comment (robotoer) [inline example.env:14]: Much clearer!

File: README.md [hunk 2]
```diff
@@ -22,13 +22,31 @@ We want to make a self-hosted workflow manager for companies and biotech labs th
...
+
+## First, prepare authentication settings
+
+At the moment, only Auth0 is supported; you will need an Auth0 account.
+
+1. Setup a react application for your auth0 tenant.  
+1. And an API.  
+1. Add the following permission scopes to the API: 
+`write:protocols`, `read:protocols`, `write:runs`, `read:runs`  
+1. Add a redirect url and allowed origin to the auth0 application: `http://localhost:3000`.
```
Comment (robotoer) [inline README.md:34]: Should these start with `1.`, `2.`, `3.`, ...?

**Review by robotoer — APPROVED** [review]
Thanks Eric! This will definitely help others get started quickly 🙂

File: README.md (hunk 2 above)
Comment (enorms) [reply README.md:34]: You mean why did I use `1.` for each?

**Comment by enorms** [comment]
Just noticed `none` is specified in `config.ts`; I can remove it from there if you want since it did not seem to function and we said `auth0` should be used.

**Pull request merged**